subsys/bluetooth/audio/has.c

```diff
@@ -34,11 +34,86 @@ static ssize_t read_features(struct bt_conn *conn, const struct bt_gatt_attr *at
 				 sizeof(has.features));
 }
 
+#if CONFIG_BT_HAS_PRESET_COUNT > 0
+static uint8_t handle_control_point_op(struct bt_conn *conn, struct net_buf_simple *buf)
+{
+	const struct bt_has_cp_hdr *hdr;
+
+	hdr = net_buf_simple_pull_mem(buf, sizeof(*hdr));
+
+	BT_DBG("conn %p opcode %s (0x%02x)", conn, bt_has_op_str(hdr->opcode), hdr->opcode);
```

Comment:
This is again one of those places where "conn" must be cast to (void *), I believe.
(Ditto elsewhere.)

Reply:
done